app/src/main/java/com/kickstarter/ui/helpers/ManagePledgeMenuOptionsFactory.kt

```diff
@@ -19,12 +19,25 @@ fun createManagePledgeMenuOptions(
     ffClient: FeatureFlagClientType
 ): ManagePledgeMenuOptions {
     val backing = project.backing()
-
     val isBackingStatusPreAuth = backing?.status() == Backing.STATUS_PREAUTH
 
-    val isEditPledgeAllowed = ffClient.getBoolean(FlagKey.ANDROID_PLOT_EDIT_PLEDGE)
+    val isFeatureFlagOn = ffClient.getBoolean(FlagKey.ANDROID_PLOT_EDIT_PLEDGE)
+    val isPlotProject = project.isPledgeOverTimeAllowed() == true // If the project allows pledge over time, it is considered a plot project.
+    val isPledgeOverTime = !backing?.paymentIncrements.isNullOrEmpty() // If the backing has payment increments, it is considered a pledge over time.
+    val showEditPledge = when {
+        isFeatureFlagOn && (isPlotProject || isPledgeOverTime) -> project.isLive && !isBackingStatusPreAuth
+        else -> false
+    }
```

Comment:
Same idea here - we want to display the **"Edit pledge"** option for all PLOT projects when the feature flag is **ON**. It shouldn't matter whether the pledge is a **Pledge Over Time** or a standard one ("Pledge in full"). In all cases, if the project meets the criteria, the option should be shown.

```suggestion
    val showEditPledge = when {
        isFeatureFlagOn && isPlotProject -> project.isLive && !isBackingStatusPreAuth
        else -> false
    }
```